### PR TITLE
RA-923:Manage Extension page UI

### PR DESCRIPTION
**Pull request by HerbertYiga**
For this https://issues.openmrs.org/browse/RA-923 ticket,i have added getExtensions() in the service so as it is used for calling the available extensions within the reference application.To use this method,changes have to be made in the reference application

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@dkayiwa   accoding to the ci builds here,this test is failing  yet it runs well locally with mvn clean install
(testGetAllEnabledExtensionsShouldCorrectlyHandleNegatedFeatureToggles) What could be the cause?

**Comment by HerbertYiga** [comment]
@mogoodrich   am requesting to help me look through my comments on this ticket !!

**Comment by mozzy11** [comment]
@HerbertYiga have you seen the Travis failure??
And also you need to squash commits into one as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java [hunk 1]
```diff
@@ -166,6 +175,7 @@ public void testGetAllEnabledExtensionsShouldCorrectlyHandleNegatedFeatureToggle
         assertEquals(2, extensionPoints.size());
         assertEquals("ext4", extensionPoints.get(0).getId());
         assertEquals("ext1", extensionPoints.get(1).getId());
+		deleteAllData();
```
Comment (mozzy11) [inline api/src/test/java/org/openmrs/module/appframework/service/AppFrameworkServiceImplTest.java]: what was the porporse of that?

**Pull request closed without merging**